miasm2/expression/simplifications_common.py

```diff
@@ -1375,6 +1391,51 @@ def simp_cond_sign_bit(_, expr):
     return ExprCond(cond, expr.src1, expr.src2)
 
 
+def simp_cond_add(expr_s, expr):
+    """
+    (a+b)?X:Y => (a == b)?Y:X
```

Comment:
There is a similar one for `^`, and I think we sometimes use the XOR form. It would be great to also have it transformed

Reply:
done